science/gmtsar-data/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar-data
+version             1.0
+categories          science
+platforms           darwin
+license             GPL-3
+
+maintainers         ucsd.edu:dsandwell \
+                    ucsd.edu:xix016 \
+                    ucsd.edu:jdesanto \
+                    openmaintainer
+
+supported_archs     noarch
+
+description         Optional orbits for GMTSAR
+long_description    Orbit solutions for the ERS and ENVISAT satellites, required to \
+                    process SAR data from these platforms.
+
+homepage            https://topex.ucsd.edu/gmtsar/
+master_sites        ftp://topex.ucsd.edu/pub/gmtsar/
+
+extract.suffix      .tar
+extract.cmd         tar
+extract.pre_args    -xvf
+
+distname            ORBITS
+distfiles           ${distname}${extract.suffix}
+
+checksums           rmd160  084f5636f7f80e1ff9536368cd2953366f3ccab7 \
+                    sha256  35e49d6c51c461566ac7bcaff5ead3243b89957d99f02eca2d969b8cf49396bf \
+                    size    6246106112
+
+extract.dir         ${workpath}/ORBITS
+
+use_configure       no
+
+build               {}
+
+destroot {
+    set datadir ${prefix}/share/${name}
+    file mkdir "${destroot}${datadir}"
```

Comment:
Quotes are unnecessary here. Portfiles are Tcl files, not shell scripts. Quoting would be necessary in a shell script to accommodate any possible spaces in the variable values, but Tcl does not have that same problem.

Reply:
Shouldn't `xinstall -d` be used instead?

`xinstall -d ${destroot}${datadir}`